lib/initial_app.dart

```diff
@@ -11,18 +13,31 @@ class MyApp extends StatelessWidget {
   // This widget is the root of your application.
   @override
   Widget build(BuildContext context) {
-    return MultiBlocProvider(providers: [
-      BlocProvider<AuthenticationBloc>(
-        create: (context) => AuthenticationBloc(),
-      )
-      // TODO add your bloc creation here
-    ], child: MaterialApp(
+    return MaterialApp(
       theme: lightThemeData,
-      home: HomePageView(),
+      home: MultiBlocProvider(
```

Comment:
MultiBlocProvider needs to wrap out side of MaterialApp so that all Blocs provided in this provider can be accessed by any screen in the app

Reply:
fixed in the next commit.